build.gradle

```diff
@@ -369,7 +375,12 @@ subprojects { subproject ->
     }
 
     implementation group: 'org.springframework', name: 'spring-jms', version: '5.3.23'
-
+    /*
+    Use actual implementation of xerces 2.12.2 instead of the one bundled with Java.  The version in openJDK contains
+    a bug in XMLSchemaValidator where cvc-complex-type.4 errors are reported twice.  This dependency can be removed
+    when the bug is fixed.
```

Comment:
We need to think of these, do we really need to add in both subprojects and allprojects. if we add this in allprojects that would be sufficient I guess. can we raise a JIRA to do some analysis on this

Reply:
Created SDT-87 (https://tools.hmcts.net/jira/browse/SDT-87) for this task.